api/fhda_api_1.1.yaml

```diff
@@ -0,0 +1,382 @@
+swagger: '2.0'
+info:
+  description: Initial API for FHDA
+  version: "1.1.1"
+  title: FHDA API
+
+  license:
+    name: Apache 2.0
+    url: http://www.apache.org/licenses/LICENSE-2.0.html
+
+# tags are used for organizing operations
+tags:
+- name: admins
+  description: Secured Admin-only calls
+
+paths:
+  /course_by_quarter:
```

Comment:
similarly, would `/api/course/get` be better? and you can use something like `/api/course/get?quarter=2020winter`